qa/rpc-tests/wallet_shieldingcoinbase.py

```diff
@@ -153,18 +154,10 @@ def run_test (self):
         assert_equal("Invalid parameter, spending key for address does not belong to wallet" in errorString, True)
 
         # Verify that debug=zrpcunsafe logs params, and that full txid is associated with opid
-        logpath = self.options.tmpdir+"/node0/regtest/debug.log"
         logcounter = 0
```

Comment:
This line can also be removed (sorry for missing it before).

Reply:
Apologies. My editor was marking the variable as unused ... This is now added as part of the last commit https://github.com/zcash/zcash/pull/4386/commits/2707e6ff6c14dfba715e2ea6451aa06b3cfc369f